lib/ex_backend/workflow/workflow_step.ex

```diff
@@ -67,6 +67,10 @@ defmodule ExBackend.WorkflowStep do
     ExBackend.Workflow.Step.AcsSynchronize.launch(workflow, step)
   end
 
+  defp launch_step(workflow, "asp_process", step) do
+    ExBackend.Workflow.Step.AspProcess.launch(workflow, step)
+  end
```

Comment:
You've missed to add restart step on case of error.
It's in workflow event controler

Reply:
Done in 40bf4a3